### PR TITLE
Cleanup logs for production deployments

### LINKED ISSUE #797: [TASK] Clean up console logs for `staging` and `main` deployments for bridge and stats
## Overview

When users open the browser console they should not be presented with console logs, or react warnings/errors. We need to go through these and clean them up. 

Ideally, we use a specific logger target where when we are developing can spill logs to the console and when we go to deploy they are not present.

### PR DESCRIPTION
**Pull request by AhmedKorim**
## Summary of changes
- 

### Reference issue to close (if applicable)
- Closes #797 

-----
### Code Checklist 

- [ ] Tested
- [ ] Documented

### CODE REVIEW

File: apps/bridge-dapp/src/components/Header/WalletButton.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const WalletButton: FC<{ account: Account; wallet: WalletConfig }> = ({
         message: 'All notes are cleared',
       });
     } catch (error) {
-      console.log('Error inside clear data', error);
+      logger.error('Error inside clear data', error);
```
Comment (AtelyPham) [inline apps/bridge-dapp/src/components/Header/WalletButton.tsx:120]: `React Hook useCallback has a missing dependency: 'logger'`

File: apps/bridge-dapp/src/components/Header/WalletButton.tsx [hunk 2]
```diff
@@ -150,8 +150,8 @@ export const WalletButton: FC<{ account: Account; wallet: WalletConfig }> = ({
   // TODO: Implement a function when user click on the new notes link
   // on the notification
   const handleNewNotes = useCallback(async (notes: Note[]) => {
-    console.log('Handle new notes: ', notes);
-    console.warn('New notes function is not implemented yet');
+    logger.info('Handle new notes: ', notes);
+    logger.warn('New notes function is not implemented yet');
```
Comment (AtelyPham) [inline apps/bridge-dapp/src/components/Header/WalletButton.tsx:154]: `React Hook useCallback has a missing dependency: 'logger'.`

File: libs/webb-ui-components/src/provider/index.tsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ const initialContext: IWebbUIContext = {
   customMainComponent: undefined,
   notificationApi,
   setMainComponent: noop,
+  logger: LoggerService.get('app'),
```
Comment (AtelyPham) [inline libs/webb-ui-components/src/provider/index.tsx:17]:
```jsx
logger: LoggerService.get('webb-ui-components'),
```

**Review by AtelyPham — COMMENTED** [review]
I think we should name the logger based on the libs/apps that initialize the logger.

File: apps/bridge-dapp/src/containers/TransferContainer/TransferConfirmContainer.tsx [hunk 4]
```diff
@@ -7,6 +7,9 @@ import { calculateTypedChainId, ChainType } from '@webb-tools/sdk-core';
 import { TransferConfirm, useWebbUI } from '@webb-tools/webb-ui-components';
 import { forwardRef, useCallback, useEffect, useMemo, useState } from 'react';
 import { TransferConfirmContainerProps } from './types';
+import { LoggerService } from '@webb-tools/app-util';
+
+const logger = LoggerService.get('ui');
```
[the remark this answers is not in the document]
Comment (AhmedKorim) [reply apps/bridge-dapp/src/containers/TransferContainer/TransferConfirmContainer.tsx:12]: Yeah added this to the UI context

**Pull request closed without merging**